cli/cmd/errors.go

```diff
@@ -85,57 +82,34 @@ func ErrorInvalidProvider(providerStr string) error {
 	})
 }
 
-func ErrorNotSupportedInLocalEnvironment() error {
-	return errors.WithStack(&errors.Error{
-		Kind:    ErrNotSupportedInLocalEnvironment,
-		Message: "this command is not supported in local environment",
-	})
-}
-
-func ErrorLocalEnvironmentCantUseClusterProvider(provider types.ProviderType) error {
-	return errors.WithStack(&errors.Error{
-		Kind:    ErrLocalEnvironmentCantUseClusterProvider,
-		Message: fmt.Sprintf("the environment named \"local\" cannot be configured to point to a cortex cluster in %s", provider),
-	})
-}
-
 func ErrorCommandNotSupportedForKind(kind userconfig.Kind, command string) error {
 	return errors.WithStack(&errors.Error{
 		Kind:    ErrCommandNotSupportedForKind,
 		Message: fmt.Sprintf("the `%s` command is not supported for %s kind", command, kind),
 	})
 }
 
-func ErrorEnvironmentNotFound(envName string) error {
+func ErrorEnvironmentNotSet() error {
 	return errors.WithStack(&errors.Error{
-		Kind:    ErrEnvironmentNotFound,
-		Message: fmt.Sprintf("unable to find environment named \"%s\"", envName),
+		Kind:    ErrEnvironmentNotSet,
+		Message: fmt.Sprintf("no default environment could be found and no environment was provided; run `cortex env default` to configure a default environment, or pass the environment name by using the `-e/--environment` flag"),
```

Comment:
Add an error if there are no environments in the CLI, which says that you need to configure an env or create a cluster